float32 position_x0
float32 position_y0
bool lidar0_update

float32 position_x1
float32 position_y1
bool lidar1_update

float32 position_x2
float32 position_y2
bool lidar2_update

float32 position_x3
float32 position_y3
bool lidar3_update

float32 position_x4
float32 position_y4
bool lidar4_update

float32 position_x5
float32 position_y5
bool lidar5_update